wrappers/aws/ses.go

```diff
@@ -0,0 +1,143 @@
+package thundraaws
+
+import (
+	"encoding/json"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/thundra-io/thundra-lambda-agent-go/application"
+	"github.com/thundra-io/thundra-lambda-agent-go/config"
+	"github.com/thundra-io/thundra-lambda-agent-go/constants"
+	"github.com/thundra-io/thundra-lambda-agent-go/tracer"
+)
+
+type sesIntegration struct{}
+type sesData struct {
+	Data string
+	Charset string
+}
+type sesBody struct {
+	Text sesData
+	Html sesData
+}
+type sesParams struct {
+	Source string
+	Destination []string
+	Subject sesData
+	Body sesBody
+	Template string
+	TemplateArn string
+	TemplateData string
+}
+type sesSendEmailParams struct {
+	Source string
+	Destination struct{
+		ToAddresses []string
+	}
+	Message struct{
+		Body sesBody
+		Subject sesData
+	}
+}
+type sesSendRawEmailParams struct {
+	Source string
+	Destinations []string
+	RawMessage sesData
+}
+type sesSendTemplatedEmailParams struct {
+	Source string
+	Destination struct{
+		ToAddresses []string
+	}
+	Template string
+	TemplateArn string
+	TemplateData string
+}
+
+func (i *sesIntegration) getSesInfo(r *request.Request) *sesParams {
+	fields := &sesParams{}
+	operationName := i.getOperationName(r)
+	if operationName == "SendEmail" {
```

Comment:
When all the if-else blocks checking for the same variable's value, using `switch-case` instead might be better